data/saas/config/power_reviews_config.yml

```
@@ -0,0 +1,44 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Power Reviews
+  type: power_reviews
```

Let's match this to the way they spell their name PowerReviews (with no space)
```suggestion
  name: PowerReviews
  type: powerreviews
```
Make sure you change the file names and any usages of the type (`powerreviews`) and name (`PowerReviews`)

---

I'll do this change, but im slightly worried that it's going to negatively improve readability, since its all lowercase